### ISSUE TITLE
[Feature Request] Deploy Major Releases to Github

### ISSUE THREAD

**Issue by snowden42** [SuperMarcus/NineAnimator#118]
Not sure how helpful this is since it will only be useful to Jailbroken users, but it’s a pretty simple best practice that we could adopt. The Travis configuration should be updated to deploy built, tagged, major releases to Github.

Relevant documentation is here: https://docs.travis-ci.com/user/deployment/releases/

Again, because these would be unsigned (I think, no expert on the iOS ecosystem) this would really only be useful for a small segment of users. But could be helpful for testing purposes and for sanity checking, etc. The same CI could also be used to build downloadable snapshot builds.

**Comment by snowden42** [comment]
Looks like you do this by hand. Stupidly the mobile view of GitHub makes it hard to see releases. I’ve updated the issue to highlight that this Issue is about automating this.

**Comment by snowden42** [comment]
Travis can be set up to sign the IPA’s but there’s some work involved.

**Comment by mmshivesh** [comment]
Can this automation export a Mac catalyst version on commits?

**Comment by SuperMarcus** [comment]
@mmshivesh yes. However, we'll be setting this up in such a way that the catalyst builds are not likely to be signed and/or notarized.

**Comment by mmshivesh** [comment]
Oh yeah, I understand, I'd love to take a look at automating this using GitHub actions and contributing a PR
